packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/cancel-deal.jsx

```diff
@@ -40,14 +56,15 @@ const CancelDeal = ({
                         margin={2}
                         message={
                             <PopoverMessageCheckbox
-                                defaultChecked={!should_show_cancellation_warning}
+                                defaultChecked={is_do_not_show_selected}
                                 message={localize(
                                     'Take profit and/or stop loss are not available while deal cancellation is active.'
                                 )}
                                 name='should_show_cancellation_warning'
-                                onChange={() => toggleCancellationWarning()}
+                                onChange={onPopoverCheckboxChange}
                             />
                         }
+                        onBubbleClose={onPopoverClose}
```

Comment:
This popover is duplicated in `toggle-positions-drawer-dialog.jsx`. We should add these changes there too or we can find a way to share this popover in `toggle-positions-drawer-dialog.jsx` and `cancel-deal.jsx`

Reply:
I added the same logic to that component too.